### ISSUE TITLE
stopping ovs-albaproxy service shouldn't stop the voldrvr

### ISSUE THREAD

**Issue by dejonghb** [openvstorage/framework#1149]
After a _**service ovs-albaproxy_vp1 stop**_ it was noted the voldrvr also gets stopped

```
OVS non-running processes
=========================

ovs-albaproxy_vp1          inactive
ovs-volumedriver_vp1       inactive
```
Restarting the abla proxy also restarts the voldrvr (if that does not fail due to a stale fuse mount if some other process holds the mountpoint busy) but the voldrvr shouldn't have been bothered.

**Comment by wimpers** [comment]
Duplicate of https://github.com/openvstorage/framework/issues/684 ?

**Comment by dejonghb** [comment]
The #684 seems to be the other way around where alba proxy suffers if the voldrvr has problems. Here the voldrvr gets stopped when the alba proxy is stopped, which should not happen.

**Comment by khenderick** [comment]
But what should the volumedriver do without running proxy? It will start giving I/O errors anyway.

**Comment by dejonghb** [comment]
Should stopping/starting (aka restarting) the proxy also restart the voldrvr?

Restarting the voldrvr will fail if its mountpoint is busy (some shell chdir'd into it) and cannot be unmounted/remounted. No I/O activity on the otherwise idle system...

**Comment by redlicha** [comment]
Expected behaviour:
- start the alba proxy (if not running already) on voldrv start
- optionally stop the alba proxy on voldrv stop
- don't start/stop voldrv on alba proxy start/stop

**Comment by khenderick** [comment]
Fixed by #1186 and openvstorage/alba-asdmanager#164, packaged in openvstorage-2.7.6-rev.4333.10879df and openvstorage-sdm-1.6.6-rev.451.50f2791

**Comment by kinvaris** [comment]
PASSED:
* Stopped the `ovs-albaproxy_myvpool01` service
* Checked `ovs monitor services`

```
OVS running processes
=====================

ovs-arakoon-config                  active
ovs-arakoon-mybackend-abm           active
ovs-arakoon-mybackend-global-abm    active
ovs-arakoon-mybackend-global-nsm_0  active
ovs-arakoon-mybackend-nsm_0         active
ovs-arakoon-mybackend02-abm         active
ovs-arakoon-mybackend02-nsm_0       active
ovs-arakoon-ovsdb                   active
ovs-arakoon-voldrv                  active
ovs-dtl_myvpool01                   active
ovs-scheduled-tasks                 active
ovs-support-agent                   active
ovs-volumedriver_myvpool01          active
ovs-volumerouter-consumer           active
ovs-watcher-config                  active
ovs-watcher-framework               active
ovs-watcher-volumedriver            active
ovs-webapp-api                      active
ovs-workers                         active


OVS non-running processes
=========================

ovs-albaproxy_myvpool01             inactive
```
